src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/EnvelopeControllerTest.java

```diff
@@ -101,6 +102,9 @@
     @Mock
     private DocumentManagementService documentManagementService;
 
+    @Mock
+    private OcrValidator ocrValidator;
```

Comment:
Given OCR is a part of the metadata file, I wonder if OCR validator should be a dependency of envelope validator.

Reply:
all methods in EnvelopeValidator are currently static :/

Reply:
One reason to not make static tools, to remember.